### ISSUE TITLE
Fix N + 1 for Refinery::Blog::Post.live

### ISSUE THREAD

**Issue by lanzhiheng** [refinery/refinerycms-blog#502]
Version: refinerycms-blog (4.0.0)

I encounter the N + 1 query when using `Refinery::Blog::Post.live`

![image](https://user-images.githubusercontent.com/8858472/51082497-d88e5e00-1742-11e9-842e-858d22db3cde.png)

Look's good if using `Refinery::Blog::Post.includes(:translations).live`

![image](https://user-images.githubusercontent.com/8858472/51082505-070c3900-1743-11e9-8bb8-dab146b1d8d0.png)

**Comment by bricesanchez** [comment]
Hi @lanzhiheng ! Could you provide a pull request?

**Comment by lanzhiheng** [comment]
@bricesanchez No, I am sorry that I forgot, I will provide a pull request later. 😢

**Comment by lanzhiheng** [comment]
@bricesanchez  Each post's record will need the translation data. so If I avoid the N + 1 query by using `includes` It will cause a very long query statement. like below

![image](https://user-images.githubusercontent.com/8858472/51184809-5c497580-1910-11e9-9001-76fc6cef51c7.png)

I think It is so hard to review by developer. May be the N + 1 version (just using `joins`) is better than this. I am not sure should I change it to `includes`? Can you give me some advices?

**Comment by parndt** [comment]
Normally I expect joins to produce only a single query; what's the peformance difference you're seeing?

**Comment by lanzhiheng** [comment]
@parndt 

![image](https://user-images.githubusercontent.com/8858472/51433676-f8fe7100-1c8a-11e9-87c1-b75addd98896.png)

I am not very sure which is better. In my opinion, I expect joins too.  😿  Can you discuss it with @bricesanchez .
